packages/grafana-ui/src/components/Collapse/Collapse.tsx

```diff
@@ -142,15 +142,25 @@ export const Collapse = ({
   const panelClass = cx([style.collapse, className]);
   const loaderClass = loading ? cx([style.loader, style.loaderActive]) : cx([style.loader]);
   const headerClass = collapsible ? cx([style.header]) : cx([style.headerCollapsed]);
```

Comment:
Also - looking at how `cx` is being used I think we can skip it for all single value styles. If I'm not missing something obvious. I know that it might not be in scope for this PR but worth considering to update it any way.